.travis.yml

```diff
@@ -5,3 +5,4 @@ bundler_args: --without development staging production
 sudo: false
 before_script:
   - bundle exec rake db:test:prepare
+  - bundle exec rake db:migrate RAILS_ENV=test
```

Comment:
Change this workaround to do the proper thing for Rails 4.1+. http://stackoverflow.com/questions/17150529/why-am-i-asked-to-run-rake-dbmigrate-rails-env-test